protocol/synthetix/subgraph/tests/event-factories.ts

```diff
@@ -600,29 +663,115 @@ export function createVaultLiquidationEvent(
   newVaultLiquidationEvent.logIndex = BigInt.fromI32(logIndex);
   newVaultLiquidationEvent.parameters = [];
   newVaultLiquidationEvent.parameters.push(
-    new ethereum.EventParam('poolId', ethereum.Value.fromUnsignedBigInt(poolId))
+    new ethereum.EventParam('poolId', ethereum.Value.fromI32(poolId))
   );
   newVaultLiquidationEvent.parameters.push(
-    new ethereum.EventParam('collateralType', ethereum.Value.fromAddress(collateralType))
+    new ethereum.EventParam(
+      'collateralType',
+      ethereum.Value.fromAddress(Address.fromString(collateralType))
+    )
   );
   const tupleArray = changetype<ethereum.Value>([
-    ethereum.Value.fromSignedBigInt(debtLiquidated),
-    ethereum.Value.fromSignedBigInt(collateralLiquidated),
-    ethereum.Value.fromSignedBigInt(amountRewarded),
+    ethereum.Value.fromSignedBigInt(BigInt.fromI64(debtLiquidated)),
+    ethereum.Value.fromSignedBigInt(BigInt.fromI64(collateralLiquidated)),
+    ethereum.Value.fromSignedBigInt(BigInt.fromI64(amountRewarded)),
   ]);
   const tuple = changetype<ethereum.Tuple>(tupleArray);
   const tupleValue = ethereum.Value.fromTuple(tuple);
   newVaultLiquidationEvent.parameters.push(new ethereum.EventParam('liquidationData', tupleValue));
   newVaultLiquidationEvent.parameters.push(
     new ethereum.EventParam(
       'liquidateAsAccountId',
-      ethereum.Value.fromSignedBigInt(liquidateAsAccountId)
+      ethereum.Value.fromSignedBigInt(BigInt.fromI64(liquidateAsAccountId))
     )
   );
   newVaultLiquidationEvent.parameters.push(
-    new ethereum.EventParam('sender', ethereum.Value.fromAddress(sender))
+    new ethereum.EventParam('sender', ethereum.Value.fromAddress(Address.fromString(sender)))
   );
   newVaultLiquidationEvent.block.timestamp = BigInt.fromI64(block['timestamp']);
   newVaultLiquidationEvent.block.number = BigInt.fromI64(block['blockNumber']);
   return newVaultLiquidationEvent;
 }
+
+export function createMarketRegisteredEvent(
```

Comment:
Instead pf moving all into one file its the opposite: split from one big file into separate per generator